src/ngx_http_lua_time.c

```diff
@@ -272,6 +272,77 @@ ngx_http_lua_ffi_time(void)
 {
     return (long) ngx_time();
 }
+
+
+void
+ngx_http_lua_ffi_update_time(void)
+{
+    ngx_time_update();
+}
+
+
+void
+ngx_http_lua_ffi_today(unsigned char *buf)
```

Comment:
Better use `u_char` instead of `unsigned char` here? There are other similar places.